package.json

```diff
@@ -99,7 +99,7 @@
     "assert": "^2.0.0",
     "async-lock": "^1.1.2",
     "base-x": "^4.0.0",
-    "biggystring": "^3.0.2",
+    "biggystring": "^4.0.0",
```

Comment:
yarn.lock wasn't updated.

Reply:
This version is already in yarn.lock, so no change needed.